.travis.yml

```diff
@@ -7,6 +7,10 @@ dist: xenial
 
 language: c
 
+arch: 
+  - amd64
+  - ppc64le
```

Comment:
Above changes will break the build matrix on travis. Try enabling ppc64le jobs by adding them under `matrix` in .travis.yml.

Reply:
Sorry if I was not clear with the changes. I meant something like this under `matrix: include`
```
- compiler: gcc
  env: MODE=build-gcc
  arch: ppc64le
````

Reply:
Thanks Siddesh !!